src/components/shared/tests/Button.spec.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { shallow } from "enzyme";
+
+import CloseButton from "../Button/Close";
+import CommandBarButton from "../Button/CommandBarButton";
+import PaneToggleButton from "../Button/PaneToggle";
+
+describe("CloseButton", () => {
+  const spy = jest.fn();
+  const tooltip = "Close something";
+  const wrapper = shallow(
+    <CloseButton buttonClass="error" handleClick={spy} tooltip={tooltip} />
+  );
+  test("renders", () => expect(wrapper).toMatchSnapshot());
+  wrapper.find("div").simulate("click");
+  test("handleClick is called", () => expect(spy).toHaveBeenCalled());
+  test("tooltip is passed through", () =>
+    expect(wrapper.find("div").prop("title")).toEqual(tooltip));
+});
+
+describe("CommandBarButton", () => {
+  test("renders", () => {
+    expect(
+      shallow(
+        <CommandBarButton className="resume" pressed={false}>
+          <img />
+        </CommandBarButton>
+      )
+    ).toMatchSnapshot();
+  });
+});
+
+describe("PaneToggleButton", () => {
+  const spy = jest.fn();
+  const wrapper = shallow(
+    <PaneToggleButton collapsed={false} handleClick={spy} position="start" />
+  );
+  test("renders start", () => expect(wrapper).toMatchSnapshot());
```

Comment:
why test and not `it`

Reply:
I heard some rumblings of `it` being deprecated by Jest and they don't directly mention it as part of their global API. http://facebook.github.io/jest/docs/en/api.html

Reply:
Got it. How about switching to `it` here to be consistent and open an issue to discuss moving to `test`...